gordo_components/cli/cli.py

```diff
@@ -41,19 +40,26 @@
 
 @click.group("gordo-components")
 @click.version_option(version=__version__, message=__version__)
-def gordo():
+@click.option(
+    "--log-level",
+    type=str,
+    default="INFO",
+    help="Run workflow with custom log-level.",
+    envvar="GORDO_LOG_LEVEL",
+)
+def gordo(log_level):
```

Comment:
I think at this level, to include each subcommand(group) you'd pass `@click.pass_context` or does it already magically work? I see the tests pass, are we waiting for anything else?